### DISCUSSION TITLE
License of the generated geometry?

### DISCUSSION THREAD

**Discussion by iamsenseiken** [Le0Michine/FusionGridfinityGenerator#58] [Q&A]
The code is under a non-com share-alike, and I absolutely respect that. I am just curious how that interacts with the output geometry. For instance, if I incorporated this output into a file I intend to sell (obviously not just the raw gridfinity stuff).

**Comment by Le0Michine** [comment]
Hi, generally SW license doesn't apply to its output (eg word document isn't subject to the license of the MS Office Word). While I would appreciate my work being credited I don't think the user has any obligation to do so in this case. So technically you should be able to do whatever with the output. At least to my limited knowledge
